app/src/main/java/org/jellyfin/androidtv/ui/browsing/StdBrowseFragment.java

```diff
@@ -126,10 +126,10 @@ protected void refreshRows() {
         new Handler().postDelayed(new Runnable() {
             @Override
             public void run() {
-                if (mActivity.isFinishing()) return;
+                if (mActivity != null && mActivity.isFinishing()) return;
```

Comment:
This should probably be `mActivity == null || mActivity.isFinishing()` to avoid NPEs.